app/services/create_next_edition_service.rb

```diff
@@ -0,0 +1,51 @@
+class CreateNextEditionService < ApplicationService
+  def initialize(current_edition:, user:, discarded_edition: nil)
+    @current_edition = current_edition
+    @user = user
+    @discarded_edition = discarded_edition
+  end
+
+  def call
+    raise "Can only create a next edition from a live edition" unless current_edition.live
+
+    current_edition.update!(current: false)
+    EditDraftEditionService.call(next_edition, user, current: true, revision: next_revision)
+    AssignEditionStatusService.call(next_edition, user: user, state: :draft)
+    next_edition.save!
+    next_edition
+  end
+
+private
+
+  attr_reader :current_edition, :user, :discarded_edition
+
+  def next_edition
+    @next_edition ||= begin
+      document = current_edition.document
+      discarded_edition || Edition.new(document: document,
+                                       number: document.next_edition_number,
+                                       created_by: user)
+    end
+  end
+
+  def next_revision
+    updater = Versioning::RevisionUpdater.new(current_edition.revision, user)
+    updater.assign(change_note: "",
+                   update_type: "major",
+                   proposed_publish_time: nil,
+                   change_history: change_history)
+    updater.next_revision
+  end
+
+  def change_history
+    if !current_edition.major? || current_edition.change_note.empty? || current_edition.first?
```

Comment:
Instead of `!current_edition.major?` you could just do `current_edition.minor?`

Reply:
With this we should be cautious about our terminology. As update_type is an enum and not a boolean we should stick to one meaning throughout. So we probably do best to decide _does change history apply to only major changes?_ or _does it not apply to minor changes?_ otherwise we end up using different words in different places to describe the same thing.

Reply:
@1pretz1 While I prefer `current_edition.minor?` for its clarity, I also take @kevindew 's point in that this, as an enum.  So, it could be 'extended' at some point - potentially, breaking this code an obscure way.  As the card states in the update criteria: _"The update type is major"_ - and if we're all ok with it - I'd prefer to leave this as `!current_edition.major?`, at least for now.  The only other resolution to this I feel would be to make `update_type` a boolean.

Reply:
Sounds good - I know having a boolean of either "major_change" or "minor_change" was unpopular at the time of building the modelling.

Reply:
@kevindew @1pretz1 - Thanks very much for your comments - sounds like we stick with the current `!current_edition.major?`.